csv_lua/sym_code.py

```diff
@@ -0,0 +1,18 @@
+class sym:
+  def __init__(self) -> None:
+    self.n = 0
+    self.at = 0
+    self.name = ""
+    self._has = dict()
+
+  def add(self,v) -> None:
+    if v!='?':
```

Comment:
I would do if v is not '?', just for readability.

Reply:
Resolved